### ISSUE TITLE
redirect rule should ignore nest.land modules

### ISSUE THREAD

**Issue by Yaikava** [denoland/deno_lint#1017]
### Lint Name
redirect

### Code Snippet

```ts
import { Kwik, KwikTable } from "https://x.nest.land/Kwik@v1.2.1/mod.ts";
```

### Expected Result

No redirect lint "warning" will be present.

### Actual Result
![image](https://user-images.githubusercontent.com/83710104/154796788-e20d6e97-03f2-4399-a96b-76d934517f32.png)

### Additional Info

[Nest.land](https://nest.land/) has these redirects for every url. I think that deno should ideally ignore this rule for all nest.land modules.

vscode extension version: `v3.10.1`
vscode version: `1.63.2`

### Version

deno 1.19.0 (release, x86_64-unknown-linux-gnu)
v8 9.9.115.7
typescript 4.5.2

**Comment by magurotuna** [comment]
The warning you ran into is not what deno_lint produces. This diagnostic was introduced to deno_lsp in https://github.com/denoland/deno/pull/13580, but as you said, this mechanism doesn't fit into nest.land unfortunately. Perhaps we need to consider how to deal with it...
